frontend/src/components/header/Header.jsx

```diff
@@ -1,66 +1,83 @@
-import SearchBar from "../searchBar/SearchBar";
+import SearchBar from '../searchBar/SearchBar';
 import React, { useState } from "react";
-import { Menu, X } from "lucide-react";
-import { Link } from "react-router-dom";
+import {LogOut, Menu, X} from "lucide-react"
+import { Link } from 'react-router-dom';
+import { UserContext } from '../../UserContext.js';
+import { useContext } from 'react';
 
 const NavLinksLogged = () => {
-  return (
+  const { updateUser } = useContext(UserContext);
+
+  const handleLogOut = async (e) => {
+    updateUser(null);
+    location.reload()
+  }
+  return(
     <>
-      <Link to="/Favorites" className="pr-8">
-        Favorites
-      </Link>
-      <Link to="/Attending" className="pr-8">
-        Attending
-      </Link>
-      <Link to="/Nearby" className="pr-8">
-        Nearby
-      </Link>
-      <Link to="/ForYou" className="pr-8">
-        For you
-      </Link>
-      <Link to="/Home" className="pr-8">
-        Home
-      </Link>
+      <Link to="/Home" className='bg-blue-700 hover:bg-blue-950 text-white font-bold py-2 px-4 rounded' onClick={handleLogOut}>Log out</Link>
+      <Link to="/Attending" className='py-2 px-4'>Attending</Link>
+      <Link to="/Nearby" className='py-2 px-4'>Nearby</Link>
+      <Link to="/ForYou" className='py-2 px-4'>For you</Link>
+      <Link to="/Home" className='py-2 px-4'>Home</Link>
+    </>
+  );
+};
+
+const NavLinksNotLogged = () => {
+  return(
+    <>
+      <Link to="/SignUp" className='py-2 px-4'>Sign Up</Link>
+      <Link to="/Login" className='py-2 px-4'>Log In</Link>
     </>
   );
 };
 
 const Header = () => {
-  const [isOpen, setIsOpen] = useState(false);
+    const [isOpen, setIsOpen] = useState(false);
+    let Logged = false;
```

Comment:
Variable naming convention, camelCase

Reply:
Also I suggest it to be named as loggedIn. Because logged can be interpreted as logging is involved.